ballerina/xmldata.bal

```diff
@@ -18,18 +18,87 @@ import ballerina/jballerina.java;
 
 const string XMLNS_NAMESPACE_URI = "http://www.w3.org/2000/xmlns/";
 const string CONTENT = "#content";
+const string ATTRIBUTE_PREFIX = "attribute_";
+
+# Defines the new name of the name.
+#
+# + value - The value of the new name
+public type NameConfig record {|
+    string value;
+|};
+
+# The annotation is used to specify the new name of the existing record name or field name according to the XML format.
+public annotation NameConfig Name on type, record field;
+
+# Defines the namespace of the XML element
+#
+# + prefix - The value of the prefix of the namespace
+# + uri - The value of the URI of the namespace
+public type NamespaceConfig record {|
+    string prefix;
+    string uri?;
```

Comment:
If user needs to add default namespace, what is the value pass for prefix?

Reply:
Don't have any default value for prefix and URL according to the open API definition and  XML definition.
Eg: Let's consider the following Open API definition:
```
components:
 schemas:
  Root
   type: object
  xml:
   prefix: ns0
   namespace: 'http://www.w3.org/' 
```
The equivalent XML format is 
```
<ns0:Root xmlns:ns0 = "http://www.w3.org/">
</ns0:Root>
```
So, users can define any prefix according to their purpose.

Reply:
How we define namespace if the XML with default namespace is like below,
```
<Root xmlns = "http://www.w3.org/">
</Root>
```

Reply:
We can define it like this.
```
components:
  schemas:
    Root:
      type: object
      xml:
        namespace: 'http://www.w3.org/'
```

Reply:
then we need to make `prefix` optional in namespace annotation right?

Reply:
and `url` mandatory?

Reply:
Yes.